compendium/CM/include/cppmicroservices/cm/ConfigurationListener.hpp

```diff
@@ -62,10 +62,10 @@ namespace cppmicroservices
                                    const ConfigurationEventType type,
                                    const std::string factoryPid,
                                    const std::string pid)
-                    : configAdmin(std::move(configAdmin))
+                    : configAdmin(configAdmin)
                     , type(type)
-                    , factoryPid(std::move(factoryPid))
-                    , pid(std::move(pid))
+                    , factoryPid(factoryPid)
+                    , pid(pid)
```

Comment:
is there benefit to having the parameter's be non-const and moving them?

Reply:
For these, we should remove the const, instead of the move

Reply:
Expensive to copy sync-params should be non-const by-value, and then moved